drift/info_parser.py

```diff
@@ -1,35 +1,52 @@
-DRIFT_FACTS = {'bios_uuid', 'display_name'}
-DRIFT_SYSTEM_ID = {'id'}
+SYSTEM_ID = 'id'
+FACT_NAMESPACE = 'inventory'
```

Comment:
We are likely to not want to restrict ourselves to a single (or any) namespace given the set of facts we'll want. If this affects your code design. If you feel this is fine for now and easy to come back and adjust as we know more then :+1:

Reply:
Just saw https://github.com/RedHatInsights/drift-backend/pull/29/files#diff-8b7de392a44280d3d4ceb46a196e26f9R44 which answers my question.